bool car_is_stop
bool battery_is_charhing